pkg/minikube/cluster/cluster.go

```diff
@@ -48,6 +48,7 @@ import (
 
 var (
 	certs = []string{"apiserver.crt", "apiserver.key"}
+	logsCmd = "docker logs origin"
```

Comment:
Will this work with OCP as well? Is the container there also named 'origin'? Or do we some smarter way to determine the right container?